src/applications/check-in/containers/withForm.jsx

```diff
@@ -1,19 +1,21 @@
 import React, { useMemo, useEffect } from 'react';
 import { useSelector } from 'react-redux';
 
-import { makeSelectForm } from '../../selectors';
+import { makeSelectForm } from '../selectors';
 
-import { useSessionStorage } from '../../hooks/useSessionStorage';
-import { useFormRouting } from '../../hooks/useFormRouting';
-import { URLS } from '../../utils/navigation/pre-check-in';
+import { useSessionStorage } from '../hooks/useSessionStorage';
+import { useFormRouting } from '../hooks/useFormRouting';
 
-const withForm = Component => {
+import { URLS } from '../utils/navigation';
+
+const withForm = (Component, options = {}) => {
+  const { isPreCheckIn } = options;
```

Comment:
I really like using the options array here for the isPreCheckIn flag. I'm wondering if it makes sense to do something similar for the  useSessionStorage hook at some point in the future.

Reply:
Probably. I have started to do that more and more and it just makes sense.